lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -344,3 +362,20 @@ def make_PUT_request(tx_audit_id: str, body: dict):
         logger.exception(e)
     finally:
         https_connection.close()
+
+
+def get_integration_testing_public_cert() -> Certificate:
+    """
+    Load the integration testing public certificate used to verify JWT signatures for POST requests.
+    In deployment environments, the certificate should be available via a lambda layer.  Mock this
+    function during unit testing.
+    """
+
+    assert NOTIFY_ENVIRONMENT != "test"
+
+    try:
+        with open("/opt/Notify_integration_testing_public.pem", "rb") as f:
+            return load_pem_x509_certificate(f.read()).public_key()
+    except (OSError, ValueError) as e:
+        logger.exception(e)
+        sys.exit("The integration testing public certificate is missing or invalid.  Cannot authenticate POST requests.")
```

Comment:
I'd like to see a generic catch and log as well. The docs don't describe all the reasons it could error, but it returns an instance of [Certificate](https://cryptography.io/en/latest/x509/reference/#cryptography.x509.Certificate), which could result in an [InvalidVersion](https://cryptography.io/en/latest/x509/reference/#cryptography.x509.InvalidVersion) error.

Reply:
changed